eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/primitive/ServiceLoaderUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 Goldman Sachs and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.api.factory.primitive;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Proxy;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.stream.Collectors;
+
+import org.eclipse.collections.api.factory.ThrowingInvocationHandler;
+
+final class ServiceLoaderUtils
```

Comment:
I think it would be fine to make org.eclipse.collections.api.factory.ServiceLoaderUtils public, instead of making a second package private copy.

Reply:
Done